packages/tabs/src/Tabs.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// hixme-ui
+import Text from '@hixme-ui/text'
+import Container from '@hixme-ui/container'
+import theme from '@hixme-ui/theme'
+
+const {
+  backgroundColors,
+} = theme
+
+const defaultStyle = {
+  transition: 'all 0.25s ease',
+  borderBottom: '4px solid white',
+  borderBottomWidth: '0',
+  marginBottom: '4px',
+}
+
+const active = {
+  transition: 'all 0.25s ease',
+  borderBottom: `4px solid ${backgroundColors.blue}`,
+  borderBottomWidth: '4px',
+}
+
+class Tabs extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      tabIndex: 0,
+    }
+  }
```

Comment:
Something @pixelfuture shared with me: you can actually forego the constructor usage and just do `state = { tabIndex: 0 }` for a bit cleaner look here.